File: 1012.py
x =input().split()
a, b, c = x
pi = 3.14159

tri = ((float(a) * float(c) )/ 2)
cir = (pi * float(c)**2)
tra = (float(c) * ((float(a) + float(b)) / 2))
qua = (float(b)**2)
ret = (float(a) * float(b))

print('TRIANGULO: {:.3f}'.format(tri))
print('CIRCULO: {:.3f}'.format(cir))
print('TRAPEZIO: {:.3f}'.format(tra))
print('QUADRADO: {:.3f}'.format(qua))
print('RETANGULO: {:.3f}'.format(ret))

File: CSharp/1012.cs
using System;
using System.Globalization;
using System.Linq;

class URI {
    static void Main(string[] args) {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        
        double[] a = Console.ReadLine().Split().Select(double.Parse).ToArray();
        double x = a[0];
        double y = a[1];
        double z = a[2];
        
        Console.WriteLine($"TRIANGULO: {x*z/2d:F3}");
        Console.WriteLine($"CIRCULO: {3.14159d*Math.Pow(z,2):F3}");
        Console.WriteLine($"TRAPEZIO: {((x+y)*z)/2d:F3}");
        Console.WriteLine($"QUADRADO: {y*y:F3}");
        Console.WriteLine($"RETANGULO: {x*y:F3}");

    }
}
